nms/app/packages/magmalte/app/components/__tests__/KPI-test.js

```diff
@@ -195,3 +234,69 @@ describe('<EnodebKPIs />', () => {
     expect(getByTestId('Transmitting')).toHaveTextContent('2');
   });
 });
+
+describe('<ServicingAccessGatewaysKPI />', () => {
+  const mockFegLteNetwork2 = {
+    ...mockFegLteNetwork,
+    federation: {feg_network_id: ''},
+    id: 'test_network2',
+  };
+  const mockFegLteNetwork3 = {...mockFegLteNetwork, id: 'test_network3'};
+  beforeEach(() => {
+    MagmaAPIBindings.getFegLte.mockResolvedValue(mockFegLteNetworks);
+    MagmaAPIBindings.getFegLteByNetworkId
+      .mockReturnValueOnce(mockFegLteNetwork)
+      .mockReturnValueOnce(mockFegLteNetwork2)
+      .mockResolvedValue(mockFegLteNetwork3);
+    MagmaAPIBindings.getLteByNetworkIdGateways
+      .mockReturnValueOnce({
+        [mockGwSt.id]: mockGwSt,
+        test_gw2: {...mockGwSt, id: 'test_gw2'},
+      })
+      .mockResolvedValue({[mockGwSt.id]: mockGwSt});
+  });
+
+  afterEach(() => {
+    axiosMock.get.mockClear();
+  });
+  const Wrapper = () => {
+    return (
+      <MemoryRouter initialEntries={['/nms/mynetwork']} initialIndex={0}>
+        <MuiThemeProvider theme={defaultTheme}>
+          <MuiStylesThemeProvider theme={defaultTheme}>
+            <Route
+              path="/nms/:networkId"
+              component={ServicingAccessGatewaysKPI}
+            />
+          </MuiStylesThemeProvider>
+        </MuiThemeProvider>
+      </MemoryRouter>
+    );
+  };
+  it('renders gateway count correctly', async () => {
+    const {getByTestId} = render(<Wrapper />);
+    await wait();
+    //first get list of feg_lte networks
```

Comment:
nit: add a space after the double slash:
```suggestion
    // first get list of feg_lte networks
```

Reply:
Done